requirements/requirements-test.txt

```
@@ -1,11 +1,11 @@
-pandas>=1.1
-flaky~=3.6
-pytest-cov==2.6.*
-pytest-timeout~=1.3
-pytest-xdist~=1.27
-pytest>=6.0
+pandas >=1.1
+pytest >7.0
+pytest-cov >4.0, <5.0
+pytest-timeout >2.0, <3.0
+pytest-xdist >3.0, <4.0
+pytest-rerunfailures >=13.0, <14.0
 ujson
 orjson
 requests
-holidays~=0.9
-matplotlib~=3.6
+holidays >=0.9
```

Always better to have an upper bound to requirements versions. I know some other don’t have it, but maybe let’s not remove them from the ones that do.

---

ok, just the 0.10 was too old
```suggestion
holidays >=0.9, <=0.43
```